src/test/mir-opt/const-promotion-extern-static/rustc.FOO.PromoteTemps.diff

```diff
@@ -61,5 +61,5 @@
       }
   }
 - 
-- alloc2 foreign static const_promotion_extern_static[317d]::[0]::X[0]
-  
+- alloc2 (const_promotion_extern_static[317d]::[0]::X[0]):
```

Comment:
No size, no align, just a `:` with nothing after it -- this looks odd.

Reply:
huh I thought I fixed this